client/package.json

```diff
@@ -61,6 +61,7 @@
     "eslint-plugin-import": "2.2.0",
     "eslint-plugin-jsx-a11y": "2.2.3",
     "eslint-plugin-react": "6.8.0",
+    "expect": "^1.20.2",
```

Comment:
How were the other tests working without this (e.g. line `530` in `root-test.js`)? :) If we need to have it explicitly mentioned, I'd just remove the `^` as I think we've decided at some point that it's better to lock the versions in Scope.